react/Modal/index.jsx

```diff
@@ -56,12 +58,12 @@ class ModalContent extends Component {
     this.scrollingContent.addEventListener('scroll', this.handleScroll, {
       passive: true
     })
-    document.body.classList.add('has-modal')
+    disableBodyScroll(document.querySelector(this.contentID))
```

Comment:
You don't need to add a new ID. You can use the already defined ref.

Something like : 
```
disableBodyScroll(this.scrollingContent)
```

To quote React documentation: 
```
 Refs are guaranteed to be up-to-date before componentDidMount or componentDidUpdate fires
```